sources/pmwebapi.go

```diff
@@ -110,7 +110,44 @@ func typeLabel(units string, pcpType string, name string) string {
 	} else {
 		name += unit
 	}
-	name = strings.Replace(name, "count / sec", "count_per_second", -1)
+	return name
+}
+
+func fixNaming(name string) string {
+	name = strings.ToLower(name)
+	switch name {
+	case "seconds":
```

Comment:
Do you need any of these blank ones? This just adds more complexity to the switchcase and more steps for the code to step through. I won't mark all of these, but I don't think we need them.

Reply:
I added these blank ones, because the way I understood it, the switch case will exit if it finds the correct case, so if seconds is written correctly in a metric I dont want it to hit the case for sec and get changed again so putting this makes the cases that are already right exit.

Reply:
Switch statements are very similar to `if..else` statements, so once it finds a case that matches, it exits the block. The only difference is when you use [fallthrough](https://github.com/golang/go/wiki/Switch#fall-through) which can still match the other cases. In your case, you do not want fallthrough (and you don't have it implemented anyway), so you can expect that if it matches one of your cases, it won't evaluate any of the others.